cli/azd/pkg/project/service_target_dotnet_containerapp.go

```diff
@@ -371,6 +372,18 @@ func (_ *containerAppTemplateManifestFuncs) UrlHost(s string) (string, error) {
 	return u.Hostname(), nil
 }
 
+func (fns *containerAppTemplateManifestFuncs) Parameter(name string) (string, error) {
+	val, found := fns.env.Config.Get("infra.parameters." + name)
```

Comment:
I'm wondering if we'll need to consider the `main.parameters.json` file as well at some point.  There are (unfortunately) multiple ways for parameters to be assigned to a bicep file.

Reply:
Not for the yaml manifest.
The yaml manifest uses the {{ .ENV.FOO }} to access environment variables. And if there is a parameter required for yaml manifest, azd set it as bicep output or secretOutput

Reply:
main.parameters.json is now also updated